File: viasoft.qualidade.rnc-frontend/apps/rnc/src/api-clients/Nao-Conformidades/Operacao-Retrabalho/model/operacao-view-output.ts
import { VsColoredTextData } from '@viasoft/components';
import { StatusProducaoRetrabalho } from './status-producao-retrabalho';

export class OperacaoViewOutput {
  public id: string;
  public numeroOperacao: string;
  public idRecurso: string;
  public descricaoRecurso: string;
  public idOperacaoRetrabalhoNaoConformidade: string;
  public status: StatusProducaoRetrabalho;
  public coloredStatus: VsColoredTextData;
}

File: QA/viasoft.qualidade.rnc.gateway/Viasoft.Qualidade.RNC.Gateway.Host/NaoConformidades/RetrabalhoNaoConformidades/OperacoesRetrabalhoNaoConformidade/Dtos/OperacaoViewOutput.cs
using System;

namespace Viasoft.Qualidade.RNC.Gateway.Host.NaoConformidades.RetrabalhoNaoConformidades.
    OperacoesRetrabalhoNaoConformidade.Dtos;

public class OperacaoViewOutput
{
    public Guid Id { get; set; }
    public string NumeroOperacao { get; set; }
    public Guid IdRecurso { get; set; }
    public string DescricaoRecurso { get; set; }
    public string CodigoRecurso { get; set; }
    public Guid IdOperacaoRetrabalhoNaoConformdiade { get; set; }
    public StatusProducaoRetrabalho Status { get; set; }
}